modules/statistics/statistics.js

```diff
@@ -54,6 +55,10 @@ function Statistics(xmpp, options) {
 Statistics.audioLevelsEnabled = false;
 
 Statistics.prototype.startRemoteStats = function (peerconnection) {
+    if (RTCBrowserType.usesORTC()) {
```

Comment:
Again, what's missing or not functioning as expected? Can it be feature detected?